src/XMLexport.cpp

```diff
@@ -1127,3 +1127,38 @@ bool XMLexport::writeKey(TKey* pT)
 
     return (isOk && (!hasError()));
 }
+
+bool XMLexport::writeScriptElement(const QString & script)
+{
+    QString localScript = script;
+    localScript.replace(QChar('\x01'), QStringLiteral("\xFFFC\x2401")); // SOH
```

Comment:
What do you think about stripping out the Object Replacement Character first before using it for our purposes? It would eliminate one possibility that could break this setup.